src/doc/src/commands/cargo-add.md

```diff
@@ -31,6 +31,11 @@ If no source is specified, then a best effort will be made to select one, includ
 
 When you add a package that is already present, the existing entry will be updated with the flags specified.
 
+Upon invocation, the enabled (`+`) and disabled (`-`) [features] of the specified dependency will be
+printed on stdout.
```

Comment:
stderr, rather than stdout (had to double check it to make sure).

Might be better to just not specify.

Reply:
Ah, thanks for checking this, you are right. I have updated this part.